gr-blocks/lib/nlog10_ff_impl.h

```diff
@@ -30,7 +30,8 @@ namespace gr {
 
     class BLOCKS_API nlog10_ff_impl : public nlog10_ff
     {
-      float  d_n;
+      float  d_n_log2_10;
+      float  d_float_min;
```

Comment:
don't add instance properties just to hold constants! (these constants can't be different for different instances of the block, right?)

Use `static const float` here, and call them `s_float_min`, for example.

But: let's be honest: rather go ahead and don't have it as class member at all. A `float min==std::numeric_limits<float>::min()` inside your `work` would achieve the same (as any compiler would see the constness of that, it's a `constexpr` starting with C++11!).